android/src/main/kotlin/com/situm/flutter/ar/situm_ar/ARController.kt

```diff
@@ -71,4 +127,13 @@ class ARController(
             arMethodCallSender.sendArGoneRequired()
         }
     }
+
+    fun updateDebugInfo() {
+        arView.updateDebugInfo(arSceneHandler.getCurrentStatusLog())
+    }
+
+
+    override fun sendARGone() {
```

Comment:
Si é unha callback, un nome máis intuitivo sería algo en plan "onArGoneRequired" ou "onArShouldStop", ou algo que nos leve a pensar que é unha resposta a un evento.